airbyte-integrations/connectors/source-gainsight-px/source_gainsight_px/manifest.yaml

```diff
@@ -109,6 +170,65 @@ definitions:
     name: "kcbot"
     primary_key: "id"
 
+  lead_events_stream:
+    $parameters:
+      path: "events/lead"
+      extractorPath: "leadEvents"
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/custom_retriever"
+    name: "lead_events"
+    primary_key: "eventId"
+
+  pageview_events_stream:
+    $parameters:
+      path: "events/pageView"
+      extractorPath: "results"
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/custom_retriever"
+    name: "pageview_events"
+    primary_key: "eventId"
+
+  segments_stream:
+    $parameters:
+      path: "segment"
+      extractorPath: "segments"
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/custom_retriever"
+    name: "segments"
+
+  segment_match_events_stream:
+    $parameters:
+      path: "events/segment_match"
+      extractorPath: "featureMatchEvents"
```

Comment:
this is not a typo. Actually stated in the docs like this
https://gainsightpx.docs.apiary.io/#reference/segmentmatchevent/v1eventssegmentmatch/get-segment-match-events